### PR TITLE
{lang}[GCCcore/8.3.0] Python v3.7.4 WIP

### PR DESCRIPTION
**Pull request by henkela**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+source_urls = ['http://www.%(namelower)s.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['d63e63e14e6d29e17490abbe6f7d17afb3db182dbd801229f14e55f4157c4ba3']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+]
+dependencies = [
+    ('bzip2', '1.0.6'),  # required for bz2 package in Python stdlib
```
Comment (Micket) [inline easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: should be using 1.0.8

File: easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.1'),
+    ('SQLite', '3.27.2'),
```
Comment (Micket) [inline easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: Should be using 3.29.0

File: easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+    ('XZ', '5.2.4'),
+    ('GMP', '6.1.2'),  # required for pycrypto
+    ('libffi', '3.2.1'),  # required for cryptography
+]
```
Comment (Micket) [inline easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: We should keep the comments for older OS'es that build OpenSSL.

File: easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+# symlink 'pip' command to 'pip3' that is included with Python installation
+# required so we can update pip to version included in extensions, using pip
+installopts = " && ln -s %(installdir)s/bin/pip3 %(installdir)s/bin/pip"
+
+# order is important!
+# package versions updated Mar 15th 2019
```
Comment (Micket) [inline easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: The entire package list should be updated.

File: easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb (hunk 3 above)
Comment (henkela) [reply easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: That was absent in 3.7.2 already. That's why it's missing

Comment (Micket) [reply easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: No? https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/Python/Python-3.7.2-GCCcore-8.2.0.eb#L29

Comment (henkela) [reply easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: interesting
in my local version it was missing actually
anyways... I'll add it again

**Comment by boegel** [comment]
We should consider picking up a patch for the problem discussed in #8771

File: easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+version = "3.7.4"
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.%(namelower)s.org/ftp/%(namelower)s/%(version)s/']
```
Comment (jose-d) [inline easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb]: I think we could go with http**s**;
` source_urls = ['https://www.%(namelower)s.org/ftp/%(namelower)s/%(version)s/']`

**Comment by boegel** [comment]
Fully updated easyconfig for Python 3.7.4 is available in #8911

**Comment by boegel** [comment]
closing this since #8911 got merged...

**Pull request closed without merging**